client/src/pages/people/Edit.tsx

```diff
@@ -117,9 +117,10 @@ const PersonEdit = ({ pageDispatchers }: PersonEditProps) => {
     }
   }
   const person = new Person(data ? data.person : {})
+  const personName = Person.militaryName(person.name)
```

Comment:
This variable can be in-lined.

Reply:
It can but I would go over the line limit and need to make a new line regardless, so I thought this was cleaner.
Do you still want me to remove the const?

Reply:
Yes please; it localises the change to just the line that is affected.